CesiumIonClient/include/CesiumIonClient/ApplicationData.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+namespace CesiumIonClient {
+
+/**
+ *  @brief An enumeration representing the values of {@link ApplicationData::applicationMode}.
+ */
+enum ApplicationMode { CesiumIon, Saml, SingleUser };
```

Comment:
I had to go look up "Saml." If it's a term of art that every programmer should know these days, then I'll try to get with the program, but I think a comment would be useful; at least, what's different about the three different modes.